docs/content/metadata/attempt/index.md

```diff
@@ -0,0 +1,16 @@
+
+# Metadata `attempt`
+
+The `attempt` metadata is an indicator of the number of times an action has been rescheduled for execution when an error occurred.
+
+* Type: `number`
+* Default: `0`
+* Read-only
+
+It is meant to be only readable from inside an handler function and to be used conjointly with the ["retry" metadata](/metadata/retry/).
```

Comment:
The property is meant to be used conjointly with the ["retry" metadata](/metadata/retry/). It is available in the handler function to know how many times an action is re-scheduled when an error occurred.